app/controllers/chargeback_controller.rb

```diff
@@ -765,14 +769,18 @@ def get_cis_all
     unless WHITELIST_INSTANCE_TYPE.include?(klass)
       raise ArgumentError, "Received: #{klass}, expected one of #{WHITELIST_INSTANCE_TYPE}"
     end
-    classtype =
+    all_of_classtype =
       if klass == "enterprise"
-        MiqEnterprise
+        MiqEnterprise.all
+      elsif klass == "ext_management_system"
+        ExtManagementSystem.all.reject { |prov| prov.is_a? ManageIQ::Providers::ContainerManager }
```

Comment:
Not commenting on the need to separate containers into it's own section ... I've never actually looked at the UI for this page before.

However, what about letting the DB do the filtering?
`ExtManagementSystem.where.not(:type => ManageIQ::Providers::ContainerManager)`